.vale.ini

```
@@ -0,0 +1,11 @@
+# This is the configuration file for the Vale linter.
+# Docs: https://docs.errata.ai/vale/config
+
+StylesPath = styles
```

```suggestion
StylesPath = .vale/styles
```
can we move? `styles`is somehow related to css styles on mos repos.